evm/p2p/kademlia.py

```diff
@@ -149,19 +158,19 @@ def split(self):
             bucket.replacement_cache.append(node)
         return lower, upper
 
-    def remove_node(self, node):
+    def remove_node(self, node: Node) -> None:
         if node not in self.nodes:
```

Comment:
Just a note that this should probably be `if node not in self` since we do define `__contains__` on this class.